app/views/sections/consignee/ConsigneeExportView.scala.html

```diff
@@ -0,0 +1,56 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.requests.DataRequest
+
+@this(
+    layout: templates.Layout,
+    formHelper: FormWithCSRF,
+    govukErrorSummary: GovukErrorSummary,
+    govukRadios: GovukRadios,
+    govukButton: GovukButton,
+    h1: components.h1,
+    continueOrExit: components.continueOrExit
+)
+
+@(form: Form[_], mode: Mode)(implicit request: DataRequest[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages("consigneeExport.title"))) {
+
+    @formHelper(action = controllers.sections.consignee.routes.ConsigneeExportController.onSubmit(request.ern, request.lrn, mode), 'autoComplete -> "off") {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form))
+        }
+
+        @govukRadios(
+            RadiosViewModel.yesNo(
+                field = form("value"),
+                legend = LegendViewModel(HtmlContent(heading)).withCssClass("govuk-fieldset__legend govuk-fieldset__legend--l")
+            )
+        )
+
+        @continueOrExit()
+    }
+}
+
+@heading = {
+    @h1("consigneeExport.heading", Some(messages("consignee.subHeading", request.lrn)), "govuk-fieldset__heading")
```

Comment:
The sub-heading needs to be removed from here. Because the a11y guidance came back and said that the caption can't be including as part of the legend.

So a separate H2 component needs to be added above the govukRadios bit.

Reply:
@MJCallahanPage Pushed https://github.com/hmrc/emcs-tfe-create-movement-frontend/pull/35/commits/cd39cd6ef926899182327b9131ec08432ae3ae6d to fix this issue.